HiP-DataStore/Controllers/ExhibitsController.cs

```diff
@@ -193,6 +195,73 @@ public IActionResult GetById(int id, DateTimeOffset? timestamp = null)
             return NoContent();
         }
 
+        [HttpGet("rating/{id}")]
```

Comment:
Although it doesn't really matter I have been told before that our routes should begin with upper case, i.e. "Rating/{id}"